src/EnergyPlus/ScheduleManager.hh

```diff
@@ -165,11 +165,12 @@ namespace ScheduleManager {
         Real64 CurrentValue;                    // For Reporting
         bool EMSActuatedOn;                     // indicates if EMS computed
         Real64 EMSValue;
+        bool UseDaylightSaving;                 // Toggles between daylight saving ON and OFF (default)
 
         // Default Constructor
         ScheduleData()
             : ScheduleTypePtr(0), WeekSchedulePointer(366, 0), Used(false), MaxMinSet(false), MaxValue(0.0), MinValue(0.0), CurrentValue(0.0),
-              EMSActuatedOn(false), EMSValue(0.0)
+              EMSActuatedOn(false), EMSValue(0.0), UseDaylightSaving(false)
```

Comment:
This is the same data structure used for _all_ schedules in EnergyPlus. So, for every schedule except the `Schedule:File` this should be true. As such I think it makes sense to default it here to `true`.